stoplightio/elements
# fix: rerenders and memoization and all that jazz

Fixes #1808 

Also decreases the amount of re-renders the app does.

To test:

1. Run demo with the following file - https://www.pfx.com.br/openapi.yaml
2. You might need to enable CORS - I used https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf?hl=en
3. Use upload form in TryIt.
4. Wait around a minute or so - changing the tabs also helped replicate the issue.
5. Expected result - file name is still present in try it (compare the behaviour with current `main`)
